### PR TITLE
New Maps & Updates

### PR DESCRIPTION
**Pull request by phellen007**
New maps and updates to all my old ones.

### CODE REVIEW

File: _maps/FortuneGateFG1/FortuneGate.yaml [hunk 1]
```diff
@@ -23,7 +23,7 @@ mapIcon: FortuneGate
 music:
   download: 
     - https://nikkums.io/cswt/BGM/FortuneGateFG1.music.zip
-    - https://drive.google.com/u/0/uc?id=1ymrEPQ50mw4yOIQYiCgo4Z8VD3lH_x5j&export=download
+    - https://drive.google.com/file/d/1tdsS_aU5A7vuHzPmHs4eTZwQjT0uGnv9/view?usp=drive_link
```
Comment (nikkiwritescode) [inline _maps/FortuneGateFG1/FortuneGate.yaml:26]: Hello! This Google Drive link works in the browser, but it doesn't work for directly downloading through tools like CSMM. So I'm going to look through your .yaml files and offer correctly-formatted links for each entry. Please update these in your files when you get a chance.

The properly-formatted download link for this file should be `https://drive.google.com/uc?export=download&id=1tdsS_aU5A7vuHzPmHs4eTZwQjT0uGnv9`

One more note for this file specifically: the music zip in Google Drive seems to have a space in its filename. Please rename it to remove the space, as those can behave unpredictably across different OSes. Thank you for all your hard work on these boards!

File: _maps/MariosSwitchboard/mariosswitchboard.yaml [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+- marioswitch5
+- marioswitch6
+- marioswitch7
+- marioswitch8
+background: grid_green
+mapIcon: switchICON
+music:
+  download:
+  - https://nikkums.io/cswt/BGM/mariosswitchboard.music.zip
+  - https://drive.google.com/file/d/1THXTKlOFLVaSuwIReoeVw0EVi0fHkLPf/view?usp=sharing
```
Comment (nikkiwritescode) [inline _maps/MariosSwitchboard/mariosswitchboard.yaml:27]: The Google Drive download link for this one should be: `https://drive.google.com/uc?export=download&confirm=1&id=1THXTKlOFLVaSuwIReoeVw0EVi0fHkLPf`

File: _maps/fortunapolis/Fortunapolis.yaml [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+salaryIncrement: 250
+maxDiceRoll: 7
+frbFile1: fortunapolis
+frbFile2: fortunapolis2
+background: bg901fixed
+mapIcon: fortunapolis
+music:
+  download:
+  - https://nikkums.io/cswt/BGM/Fortunapolis.music.zip
+  - https://drive.google.com/file/d/1bkGFZpadHsJ56twXWFQVa50TslV6jNOk/view?usp=sharing
```
Comment (nikkiwritescode) [inline _maps/fortunapolis/Fortunapolis.yaml:20]: This Google Drive download link should be: `https://drive.google.com/uc?export=download&confirm=1&id=1bkGFZpadHsJ56twXWFQVa50TslV6jNOk`

File: _maps/giantbeanstalk/giantbeanstalk.yaml [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
+  mode: Vertical
+  radius: 28
+  horizontalPadding: .95
+  verticalSquareCount: 17
+background: bg008alt
+mapIcon: gbs
+music:
+  download:
+  - https://nikkums.io/cswt/BGM/GiantBeanstalk.music.zip
+  - https://drive.google.com/file/d/1PujpGrdxuR0dF_x8xGaqJ2se3__mYjJ0/view?usp=drive_link
```
Comment (nikkiwritescode) [inline _maps/giantbeanstalk/giantbeanstalk.yaml:24]: This Google Drive download link should be: `https://drive.google.com/uc?export=download&confirm=1&id=1PujpGrdxuR0dF_x8xGaqJ2se3__mYjJ0`

File: _maps/lighthouseisland/lighthouseisland.yaml [hunk 5]
```diff
@@ -0,0 +1,189 @@
...
+frbFile1: lighthouse
+frbFile2: lighthouse2
+frbFile3: lighthouse3
+frbFile4: lighthouse4
+background: bg102alt
+mapIcon: lightICON
+music:
+  download:
+  - https://nikkums.io/cswt/BGM/lighthouseisland.music.zip
+  - https://drive.google.com/file/d/1yoEBKlOkr9pJNWev-xWZ8nuIWdHYGhU2/view?usp=sharing
```
Comment (nikkiwritescode) [inline _maps/lighthouseisland/lighthouseisland.yaml:22]: This Google Drive download link should be: `https://drive.google.com/uc?export=download&confirm=1&id=1yoEBKlOkr9pJNWev-xWZ8nuIWdHYGhU2`

File: _maps/slimeguyspaydayexpress/slimeguyspaydayexpress.yaml [hunk 6]
```diff
@@ -0,0 +1,182 @@
...
+frbFile1: slimeguyspaydayexpress1s
+frbFile2: slimeguyspaydayexpress2s
+frbFile3: slimeguyspaydayexpress3s
+frbFile4: slimeguyspaydayexpress4s
+background: bg109
+mapIcon: paydayexpress
+music:
+  download: 
+   - https://nikkums.io/cswt/BGM/slimeguyspaydayexpress.music.zip
+   - https://drive.google.com/file/d/16y8nk0E1Xjh2fH_d0xAb1zAc97A6qxI6/view?usp=drive_link
```
Comment (nikkiwritescode) [inline _maps/slimeguyspaydayexpress/slimeguyspaydayexpress.yaml:22]: This one should be `https://drive.google.com/uc?export=download&confirm=1&id=16y8nk0E1Xjh2fH_d0xAb1zAc97A6qxI6`

File: _maps/superbowserstrackway/superbowserstrackway.yaml [hunk 7]
```diff
@@ -0,0 +1,187 @@
...
+salaryIncrement: 275
+maxDiceRoll: 6
+frbFiles:
+ - superbowserstrackway
+background: forestFairies
+mapIcon: sbowsersICON
+music:
+  download:
+  - https://nikkums.io/cswt/BGM/superbowserstrackway.music.zip
+  - https://drive.google.com/file/d/1P-Tq8jVU0cuqEpHoA-6P69RvTU_b1uUR/view?usp=sharing
```
Comment (nikkiwritescode) [inline _maps/superbowserstrackway/superbowserstrackway.yaml:20]: This one should be `https://drive.google.com/uc?export=download&confirm=1&id=1P-Tq8jVU0cuqEpHoA-6P69RvTU_b1uUR`

File: _maps/thebackstreets/thebackstreets.yaml [hunk 8]
```diff
@@ -0,0 +1,186 @@
...
+frbFile1: TheBackstreets1
+frbFile2: TheBackstreets2
+frbFile3: TheBackstreets3
+frbFile4: TheBackstreets4
+background: traverseTown
+mapIcon: backstreets
+music:
+  download: 
+  - https://nikkums.io/cswt/BGM/thebackstreets.music.zip
+  - https://drive.google.com/file/d/1RwK5rUJhayHv4o-By0IpTIyW7AADd-0_/view?usp=sharing
```
Comment (nikkiwritescode) [inline _maps/thebackstreets/thebackstreets.yaml:22]: This one should be: https://drive.google.com/uc?export=download&confirm=1&id=1RwK5rUJhayHv4o-By0IpTIyW7AADd-0_

Comment (phellen007) [reply _maps/thebackstreets/thebackstreets.yaml:22]: Thanks so much for fixing those links! I'll add those corrections and resubmit the boards again soon.

File: _maps/MariosSwitchboard/mariosswitchboard.yaml (hunk 2 above)
Comment (phellen007) [reply _maps/MariosSwitchboard/mariosswitchboard.yaml:27]: I'll hold off on fixing this one since it's going to be the new Star Carnival board.

**Comment by Sophilautia** [comment]
Looks like these music tracks still need to be uploaded to s3. I can't access the s3 myself right now, so I can't fix it until I get access again.

**Comment by phellen007** [comment]
If you want to just leave Toad's Treasure hunt music as it is from the first version, that's fine with me. I think I updated it, but I actually didn't mind the music from before. So whatever is easiest on your end is fine with me.

**Pull request merged**